### ISSUE TITLE
Feature Request : Remove a member

### ISSUE THREAD

**Issue by bchooxg** [oss-apps/split-pro#159]

**Comment by guenzd** [comment]
Like in SW it should be only possible to remove a member from a group, if the member is settled - otherwise the "X" button should not be clickable.

**Comment by krokosik** [comment]
@guenzd Are you still interested in helping? The repo is now back in active development and some SQL improvements are under consideration :)

**Comment by guenzd** [comment]
@krokosik in general yes, but the project seemed quite on hold for a long time, will this change?

**Comment by krokosik** [comment]
Yes, I have been given maintainer rights and actively work on the project. However, it's only me and extra help would be very helpful as there is plenty of interesting features to implement, as well as some code refactors to do.

**Comment by guenzd** [comment]
Thanks for the info, if you happen to have a smallish sql related issue I can give it a try :)

**Comment by krokosik** [comment]
If you've worked with SQL stuff, I believe your insight would be very valuable on the DB schema revamp proposed by @FriesischScott in #234. The current codebase relies heavily on data manipulation in TypeScript which I would like to change, but also I'm afraid of messing up people's data, so another set of watchful eyes would be very helpful.
